src/Services/cdl/Debugger.js

```
@@ -108,53 +92,129 @@ class Debugger {
+     * @param {Number} position 
+     * @returns 
+     */
+    playForward (position) {
+        do {
+            position = this.cdl._getNextPosition(position);
+
+            if (position == null) {
+                return;
+            }
+
+            for (let breakpoint of this.breakpoints) {
```

ESLint

[error] 'breakpoint' is never reassigned. Use 'const' instead.

(prefer-const)